readthedocs/projects/management/commands/dump_project_remote_repo_relation.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from django.core.management.base import BaseCommand
+
+from readthedocs.projects.models import Project
+
+
+class Command(BaseCommand):
+    help = "Dump Project and RemoteRepository Relationship in JSON format"
+
+    def handle(self, *args, **options):
+        data = []
+
+        queryset = Project.objects.filter(
+            remote_repository__isnull=False,
+        ).values_list('id', 'remote_repository__json').distinct()
+
+        for project_id, remote_repository__json in queryset:
```

Comment:
```suggestion
        for project_id, remote_repository__json in queryset.iterator():
```

I think we need iterator here because we will probably have a lot of projects.

Reply:
This query takes about ~10s, just to COUNT:

```
In [2]: queryset.count()
Out[2]: 39468
```